Sofa/framework/DefaultType/src/sofa/defaulttype/RigidTypes.h

```diff
@@ -26,6 +26,7 @@
 #include <sofa/defaulttype/RigidDeriv.h>
 #include <sofa/defaulttype/RigidMass.h>
 #include <sofa/defaulttype/MapMapSparseMatrix.h>
+#include <sofa/linearalgebra/CompressedRowSparseMatrixConstraint.h>
```

Comment:
```suggestion
#include <sofa/linearalgebra/CompressedRowSparseMatrixConstraint.h>
```
Do you still need MapMapSparseMatrix.h?